### PR TITLE
docs: Mention Helm support. Fixes #1119

### LINKED ISSUE #1119: Helm support should be clear in the docs
# Summary

We don't have any documentation that mentions how you can use Helm with Argo Rollouts

# Use Cases

Open discussion is here https://github.com/argoproj/argo-rollouts/discussions/898 (support for Helm)

---

**Message from the maintainers**:

Impacted by this bug? Give it a 👍. We prioritize the issues with the most 👍.

### PR DESCRIPTION
**Pull request by kostis-codefresh**
Signed-off-by: Kostis Kapelonis <kostis@codefresh.io>

Checklist:

* [X] Either (a) I've created an [enhancement proposal](https://github.com/argoproj/argo-rollouts/issues/new/choose) and discussed it with the community, (b) this is a bug fix, or (c) this is a chore.
* [X] The title of the PR is (a) [conventional](https://www.conventionalcommits.org/en/v1.0.0/), (b) states what changed, and (c) suffixes the related issues number. E.g. `"fix(controller): Updates such and such. Fixes #1234"`.  
* [X] I've signed my commits with [DCO](https://github.com/argoproj/argoproj)
* [ ] I have written unit and/or e2e tests for my change. PRs without these are unlikely to be merged.
* [ ] My builds are green. Try syncing with master if they are not. 
* [ ] My organization is added to [USERS.md](https://github.com/argoproj/argo-rollouts/blob/master/USERS.md).

### CODE REVIEW

File: examples/helm-blue-green/README.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Blue Green
```
Comment (huikang) [inline examples/helm-blue-green/README.md:1]: Feel like this doc is more related to argocd. Would be better if we add some description of argo-rollout in the context.

**Review by huikang — COMMENTED** [review]
minor comment. Thanks.

File: examples/helm-blue-green/README.md (hunk 1 above)
Comment (kostis-codefresh) [reply examples/helm-blue-green/README.md:1]: Yes I took that example from ArgoCD. Ok I will update the Readme.

**Comment by kostis-codefresh** [comment]
@huikang I updated the readme. Let me know what you think

File: examples/helm-blue-green/README.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+Now, two versions will exist in your cluster (and each one has an associated service)
+
+```
+kubectl-argo-rollouts get rollout example-helm-guestbook
+```
+
+## Promoting the rollout
+
+To advanced the rollout and make the new version stable
```
Comment (huikang) [inline examples/helm-blue-green/README.md:38]: nit: advance

**Review by huikang — APPROVED** [review]
LGTM

File: examples/helm-blue-green/README.md (hunk 2 above)
Comment (kostis-codefresh) [reply examples/helm-blue-green/README.md]: @huikang I fixed it.

**Review by jessesuen — APPROVED** [review]
Thanks for the docs improvement @kostis-codefresh and thanks for reviewing this @huikang!

**Pull request merged**